omnigibson/simulator.py

```diff
@@ -1733,6 +1736,14 @@ def initial_rendering_dt(self):
             """
             return self._initial_rendering_dt
 
+        @property
+        def contact_sensor(self):
```

Comment:
should this live under usd_utils as a separate API singleton? or maybe part of RigidContactAPI?

Reply:
Good point! I did think about integrating it into RigidContactAPI, but went with this to get minimal changes in this PR. 

@cgokmen @ChengshuLi What are your thoughts on this?

Reply:
I think this is fine to keep here, it's just like the physx sim interface etc. that just provide a centralized interface and dont provide any additional functionality